astroid/rebuilder.py

```diff
@@ -1822,6 +1822,15 @@ def visit_try(
             return self.visit_tryexcept(node, parent)
         return None
 
+    def visit_trystar(self, node: ast.TryStar, parent: NodeNG) -> nodes.TryStar:
+        return nodes.TryStar(
+            lineno=node.lineno,
+            col_offset=node.col_offset,
+            end_lineno=getattr(node, "end_lineno", None),
+            end_col_offset=getattr(node, "end_col_offset", None),
+            parent=parent,
+        )
```

Comment:
Judging from `visit_try()` directly above this, we will need to do something very similar to populate the body, handlers, and finally.